### DISCUSSION TITLE
Is it possible to call internal API POST/PUT request?

### DISCUSSION THREAD

**Discussion by AsifArmanRahman** [encode/django-rest-framework#9009] [Question & Answer]
I'm actually trying to execute an internal call of an API endpoint. I can see a issue was raised almost 11 years ago, and some approaches were suggested here https://github.com/encode/django-rest-framework/issues/61.

But as the technique is years old, it doesn't work with the current version of the DRF. Is there any way to achieve it with the latest DRF version?

**Comment by samhirtarif** [comment]
The approach suggested is still considered a legitimate approach

Another way that I've done it is:

```python
api_response = ExampleApiView().post(request)
```
